recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,6 @@
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "11.3"                 # [osx and x86_64]
 c_compiler_version:            # [osx]
-  - 15                         # [osx]
+  - 16                         # [osx]
 cxx_compiler_version:          # [osx]
-  - 15                         # [osx]
+  - 16                         # [osx]
```

Comment:
It seems by default it picks `clang 16`. So, I think we need not mention`c_compiler_version` and `cxx_compiler_version` here.

Reply:
But may be we can mention, so that we use a fixed/pinned version at the CI.

Reply:
https://github.com/conda-forge/lfortran-feedstock/pull/61